serviceregistry-server/src/main/java/no/difi/meldingsutveksling/serviceregistry/servicerecord/ServiceRecordFactory.java

```diff
@@ -198,42 +226,77 @@ private String lookupPemCertificate(String orgnumber) {
     }
 
     @PreAuthorize("#oauth2.hasScope('move/dpi.read')")
-    public Optional<ServiceRecord> createServiceRecordForCititzen(String identifier,
-                                                                  Authentication auth,
-                                                                  String onBehalfOrgnr,
-                                                                  Notification notification,
-                                                                  boolean forcePrint) throws KRRClientException {
+    public List<ServiceRecord> createDigitalpostServiceRecords(String identifier,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this method to throw at most one checked exception instead of: no.difi.meldingsutveksling.serviceregistry.krr.KRRClientException, no.difi.meldingsutveksling.serviceregistry.krr.DsfLookupException, no.difi.meldingsutveksling.serviceregistry.service.brreg.BrregNotFoundException [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1160)